//----------------------------------------------------------------------------
// Copyright 2023, Ed Keenan, all rights reserved.
//----------------------------------------------------------------------------

syntax = "proto3";

import "FrameBucketEntry.proto";

option optimize_for = LITE_RUNTIME;

message ClipData_proto
{
	optional bytes                           pVersion  = 1;
	optional uint32                       numKeyFrames = 2;
	optional uint32                           numBones = 3;
	repeated FrameBucketEntry_proto poFrameBucketEntry = 4;
}

// -- End of File ---
